config/routes.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
```

Comment:
Shouldn't this file be in the gemspec too?

Reply:
ohh true.